plugins/zfs/zfs_list

```diff
@@ -1,21 +1,37 @@
 #!/bin/bash
-#
-# Plugin to monitor ZFS Filesystems
-# Author: Adam Michel (elfurbe@furbism.com)
-# Description:
-#  This is an extension of the zfs_fs plugin
-#  modified as a multigraph to graph all zfs
-#  filesystems it can find
-#
-# Tested on Ubuntu-14.04
-#
-# Parameters understood:
-#
-# 	config   (required)
-# 	autoconf (optional - used by munin-config)
-#
 #%# family=auto
 
+: << EOF
+=head1 NAME
+zfs_list - Plugin to monitor ZFS Filesystems
+
+This is an extension of the zfs_fs plugin
+modified as a multigraph to graph all zfs
+filesystems it can find
+
+=head1 CONFIGURATION
+plugin config:
+[zfs_list]
+env.ignore_datasets_string <Datasets to ignore; String for grep>
```

Comment:
Maybe there is a way to improve the name of this settings?
I would recommend to replace `string` with `pattern` (in order to refer to a regular expression).
Regarding `datasets`: I cannot judge, whether this is the proper term.

Reply:
changed it